pkg/manifest/manifest_loader.go

```diff
@@ -220,12 +220,19 @@ func toEnvironments(context *ManifestLoaderContext, groups []group) (map[string]
 	var errors []error
 	environments := make(map[string]EnvironmentDefinition)
 
+	groupNames := make(map[string]bool, len(groups))
+
 	for i, group := range groups {
 		if group.Name == "" {
 			errors = append(errors, newManifestLoaderError(context.ManifestPath, fmt.Sprintf("missing group name on index `%d`", i)))
-			continue
```

Comment:
I removed this continue as any error would fail the manifest-loading.
With this removed, we can collect all errors immediately instead of showing this single error to the user and maybe showing more after it is fixed. Wdyt?

Reply:
It would not 'impact' the behavior, but only some error messages, e.g. if the token is not set, the name cannot be printed